boards/common/nrf52/include/cfg_timer_default.h

```diff
@@ -43,15 +43,17 @@ static const timer_conf_t timer_config[] = {
         .bitmode  = TIMER_BITMODE_BITMODE_08Bit,
         .irqn     = TIMER2_IRQn
     },
+    /* The later timers are only present on the larger NRF52 CPUs like NRF52840
+     * or NRF52833, but not small ones like NRF52810 */
+#ifdef NRF_TIMER4
```

Comment:
```suggestion
#ifdef NRF_TIMER3
```

Reply:
Indeed, thanks, and sorry for the clutter.